src/commands/scaling/addScaleRule/GetScaleRuleTypeStep.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IWizardOptions } from '@microsoft/vscode-azext-utils';
+import { QuickPickItem } from 'vscode';
+import { ScaleRuleTypes } from '../../../constants';
+import { GetHttpConcurrentRequestsStep } from './http/GetHttpConcurrentRequestsStep';
+import { IAddScaleRuleWizardContext } from './IAddScaleRuleWizardContext';
+import { GetQueueAuthSecretStep } from './queue/GetQueueAuthStep';
+import { GetQueueAuthTriggerStep } from './queue/GetQueueAuthTriggerStep';
+import { GetQueueLengthStep } from './queue/GetQueueLengthStep';
+import { GetQueueNameStep } from './queue/GetQueueNameStep';
+
+export class GetScaleRuleTypeStep extends AzureWizardPromptStep<IAddScaleRuleWizardContext> {
```

Comment:
I think we can just call it `ScaleRuleTypeStep`. I think a promptStep is kind of assumed to be a "get" of some sort.